plugins/mcreator-core/procedures/entity_set_oxygen.json

```diff
@@ -0,0 +1,29 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    },
+    {
+      "type": "input_value",
+      "name": "air",
+      "check": "Number"
+    }
+  ],
+  "inputsInline": true,
+  "previousStatement": null,
+  "nextStatement": null,
+  "colour": 175,
```

Comment:
This procedure, for example, is now in proper category, but has color of player category. Please check other blocks too and match the colors.